shared/actions/chat.js

```diff
@@ -179,7 +179,7 @@ function selectConversation (conversationIDKey: ConversationIDKey, fromUser: boo
   return {type: 'chat:selectConversation', payload: {conversationIDKey, fromUser}}
 }
 
-function _inboxConversationToConversation (convo: ConversationLocal, author: ?string, following: {[key: string]: boolean}, metaData: MetaData): ?InboxState {
+function _inboxConversationToConversation (convo: ConversationLocal, author: ?string, following: {[key: string]: boolean}, metaData: MetaData, youCreatedConversationIDKey: ?ConversationIDKey): ?InboxState {
```

Comment:
yeah I figured that was why. It seems a little weird to have it this way though. This function just transforms data into our InboxState type, with `youCreatedConversationIDKey` you make the caller understand that if they pass the convID that equals the convID in the given `convo` then `youCreated` flag is set. 

What about having a `conversationToIDKey` function that does the null checks and stuff